runtimes/swift-5.5/Sources/App/routes.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 struct RequestValue {
     var path: String = "/usr/code"
-    var file: String = "index.swift"
+    var file: String = "Sources/index.swift"
```

Comment:
Shouldnt this value be set by the environment variable ?  rather than hardcoding it ? 
What if the developer has a different one ?

Reply:
Not sure... Tests use different entrypoint so it works, but I can't see the build process moving anything to the index file. Maybe this variable is not being used.

Reply:
This is definitely just setting the default value and it's fine. we overrite it in line 28 below.